test/1_stdlib/Inputs/PrintTestTypes.swift

```diff
@@ -0,0 +1,153 @@
+import Swift
+import Foundation
```

Comment:
It is not required to import Swift explicitly.  Why do we import Foundation?  We can't do it on Linux in the swift repository.